packages/url_launcher/url_launcher_platform_interface/lib/method_channel_url_launcher.dart

```diff
@@ -46,6 +47,7 @@ class MethodChannelUrlLauncher extends UrlLauncherPlatform {
         'enableDomStorage': enableDomStorage,
         'universalLinksOnly': universalLinksOnly,
         'headers': headers,
+        'windowName': windowName,
```

Comment:
Since this is a web-only parameter, there's no need for passing it through the Method Channel (which is only used by the Android/iOS implementations). Do not add 'windowName' to the Map that is passed to invokeMethod.